src/rxstrings.c

```diff
@@ -135,6 +135,57 @@ int CheckAndCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
   return REDISMODULE_OK;
 }
 
+/*
+* PREPEND key value
+* Prepends a value to a String key.
+* If key does not exist it is created and set as an empty string,
+* so PREPEND will be similar to SET in this special case.
+* Integer Reply: the length of the string after the prepend operation.
+*/
+int PrependCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+  if (argc != 3) {
+    return RedisModule_WrongArity(ctx);
+  }
+  RedisModule_AutoMemory(ctx);
+
+  /* Obtain key and extract arg. */
+  RedisModuleKey *key =
+      RedisModule_OpenKey(ctx, argv[1], REDISMODULE_READ | REDISMODULE_WRITE);
+  size_t argLen;
+  const char* argPtr = RedisModule_StringPtrLen(argv[2], &argLen);
+
+  /* SET if empty */
+  if (RedisModule_KeyType(key) == REDISMODULE_KEYTYPE_EMPTY) {
+    if (RedisModule_StringSet(key, argv[2]) == REDISMODULE_OK) {
+      RedisModule_ReplyWithLongLong(ctx, argLen);
+      return REDISMODULE_OK;
+    }
+    RedisModule_ReplyWithError(ctx, "PREPEND ERR: RM_StringSet failed");
```

Comment:
Please change the message to the convention by prepending (pun intended) the error message w/ "ERR", i.e. "ERR RM_StringSet failed". Similarly for line 177.